packages/cli/src/tools/copyFiles.js

```diff
@@ -64,10 +64,7 @@ function copyBinaryFile(srcPath, destPath, cb) {
   });
   readStream.on('close', () => {
     done();
-    // We may revisit setting mode to original later, however this fn is used
-    // before "replace placeholder in template" step, which expects files to be
-    // writable.
-    // fs.chmodSync(destPath, mode);
+    fs.chmodSync(destPath, mode);
```

Comment:
unrelated bugfix: while testing I noticed that `gradlew` is not executable in newly init projects, so brought chmod back (we're not copying Pods anymore anyway, so we should be fine)

Reply:
Sounds good, thanks for explaining this particular change!